app/styles/main.scss

```diff
@@ -1,97 +1,8109 @@
+    font-size: 19px;
+    font-size: 1.1875rem;
+    line-height: 1.3
+  }
+}
 
-  @include mq($until: desktop) {
-    @include nhsuk-font(19, $line-height: 1.3);
-    margin-bottom: nhsuk-spacing(2);
+@media print and (max-width: 48.0525em) {
+  .app-side-nav__list {
+    font-size: 14pt;
+    line-height: 1.3
   }
 }
 
 .app-side-nav__item {
   padding-bottom: 4px;
-  padding-top: 4px;
+  padding-top: 4px
 }
 
 .app-side-nav__item--current {
-  border-left: 4px solid $color_nhsuk-blue;
+  border-left: 4px solid #005eb8;
   margin-left: -12px;
-  padding-left: 8px;
+  padding-left: 8px
+}
 
-  .app-side-nav__link {
-    font-weight: $nhsuk-font-bold;
-  }
+.app-side-nav__item--current .app-side-nav__link {
+  font-weight: 600
 }
 
 .app-side-nav__link {
-  text-decoration: none;
+  text-decoration: none
+}
 
-  @include mq($until: desktop) {
-    text-decoration: underline;
+@media (max-width: 48.0525em) {
+  .app-side-nav__link {
+    text-decoration: underline
   }
+}
 
-  &:visited {
-    color: $color_nhsuk-blue;
-  }
+.app-side-nav__link:visited {
+  color: #005eb8
+}
+
+.app-side-nav__link:hover {
+  color: #7C2855;
+  text-decoration: underline
+}
+
+.app-side-nav__link:focus:hover,
+.app-side-nav__link:focus:visited {
+  color: #212b32
+}
+
+.app-side-nav__list--nested {
+  margin-bottom: 8px;
+  margin-top: 8px
+}
 
-  &:hover {
-    color: $nhsuk-link-hover-color;
-    text-decoration: underline;
+@media (min-width: 40.0625em) {
+  .app-side-nav__list--nested {
+    margin-bottom: 8px
   }
+}
 
-  &:focus {
-    &:hover,
-    &:visited {
-      color: $nhsuk-focus-text-color;
-    }
+@media (min-width: 40.0625em) {
+  .app-side-nav__list--nested {
+    margin-top: 16px
   }
 }
 
-.app-side-nav__list--nested {
-  @include nhsuk-responsive-margin(2, 'bottom');
-  @include nhsuk-responsive-margin(3, 'top');
+.app-side-nav__list--nested .app-side-nav__item {
+  font-weight: 400;
+  font-size: 14px;
+  font-size: .875rem;
+  line-height: 1.3;
+  list-style: none
+}
 
-  .app-side-nav__item {
-    @include nhsuk-font(16, $line-height: 1.3);
-    list-style: none;
+@media (min-width: 40.0625em) {
+  .app-side-nav__list--nested .app-side-nav__item {
+    font-size: 16px;
+    font-size: 1rem;
+    line-height: 1.3
   }
+}
 
-  .app-side-nav__link {
-    font-weight: 400;
+@media print {
+  .app-side-nav__list--nested .app-side-nav__item {
+    font-size: 14pt;
+    line-height: 1.3
   }
 }
 
+.app-side-nav__list--nested .app-side-nav__link {
+  font-weight: 400
+}
+
 .app-side-nav__list--nested .app-side-nav__item::before {
-  color: $color_nhsuk-grey-3;
+  color: #aeb7bd;
   content: '—';
   margin-left: -20px;
-  margin-right: 4px;
+  margin-right: 4px
 }
 
 .app-index-navigation {
-  @include nhsuk-responsive-padding(4, 'top');
+  padding-top: 16px
+}
+
+@media (min-width: 40.0625em) {
+  .app-index-navigation {
+    padding-top: 24px
+  }
+}
+
+/* ==========================================================================
```

Comment:
Is everything below here from the NHS.UK frontend? We don't need all these styles as we're importing the NHS.UK frontend styles at the start of this file `@import '../../node_modules/nhsuk-frontend/packages/nhsuk.scss';`

Reply:
I haven't looked into much detail why, but some of the widths and styles from the menu didn't pull through. I added these styles temporality to while I worked through the rest of the changes. 

I will now relook at the reasons why, thanks for picking up.